spec/parsers/date_string_parser_spec.rb

```diff
@@ -51,6 +51,11 @@
     # Dates should be interpretted as UK not US
     "01/11/2014" => Date.new(2014, 11, 1),
 
+    # Datetime strings with timezones (BST & GMT)
+    "2025-04-10 23:00:05 +0000" => Date.new(2025, 4, 11),
```

Comment:
Ditto here.

Reply:
sorry might have misunderstood your question - it parses it into the timezone of the app (which is currently set to London). So by context, it should be able to parse any time in any other timezones back into London, and then fetching the date for that